docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -9,10 +9,10 @@ AddressBook *-right-> "1" UniqueTagList
 UniqueTagList -[hidden]down- UniquePersonList
 UniqueTagList -[hidden]down- UniquePersonList
 
-UniqueTagList -right-> "*" Tag
+UniqueTagList -right-> "*" Allergy
```

Comment:
Have to change to `UniqueAllergyList`

Reply:
Can add one for `OwnedAppointmentList` also